master/contrib/github_buildbot.py

```diff
@@ -104,10 +105,10 @@ def render_POST(self, request):
             request.setResponseCode(OK)
             return json.dumps({"result": "pong"})
 
-        # Reject non-push, non-ping events
-        if event_type != "push":
+        # Reject events not involving code changes
+        if event_type not in ("push", "pull_request"):
```

Comment:
@yetanotherion Enum is new to Python 3.4. Wouldn't that break compatibility with older versions of python (such as 2.7)?